docs/semgrep-appsec-platform/wiz.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: wiz
+append_help_link: true
+title: Wiz
+hide_title: true
+description: "Learn how to view Semgrep findings in Wiz's Security Graph."
+tags:
+ - Wiz
+ - Semgrep AppSec Platform
+---
+
+# View Semgrep findings in Wiz's Security Graph
+
+Semgrep allows you to send high severity, high confidence Semgrep Code findings for inclusion in the Wiz Security Graph.
```

Comment:
```suggestion
Semgrep allows you to send Semgrep Code findings for inclusion in the Wiz Security Graph.
```
Technically, we are not limited to high severity, high confidence Semgrep code findings.